go.mod

```diff
@@ -15,15 +15,15 @@ require (
 	github.com/matryer/moq v0.0.0-20190312154309-6cfb0558e1bd
 	github.com/pkg/errors v0.8.1
 	github.com/prometheus/client_golang v1.4.0
-	github.com/rancher/wrangler v0.4.1
+	github.com/rancher/wrangler v0.5.4-0.20200326191509-4054411d9736
```

Comment:
Need to update this after merging https://github.com/rancher/wrangler/pull/93 with the commit from the release/v2.4 branch